rllib/algorithms/a2c/a2c.py

```diff
@@ -51,11 +53,11 @@ class A2CConfig(A3CConfig):
         >>> config.environment(env="CartPole-v1")
         >>> # Use to_dict() to get the old-style python config dict
         >>> # when running with tune.
-        >>> tune.run(
+        >>> tune.Tuner(
```

Comment:
I think `tune.run` had a certain ring to it, but going forward I think we should use

```
from ray.tune import Tuner
Tuner(...).fit()
```

instead. Not for this PR, though.

Reply:
So the reason why we often use `tune.Tuner` is so we can continue to use the rest of the `tune.xyz` scope and reduce the number of imports. Happy to switch this all in one big PR at some point if you feel strongly about this